src/progressbar/progressbar.spec.ts

```diff
@@ -0,0 +1,27 @@
+import {iit, it, ddescribe, describe, expect, injectAsync, TestComponentBuilder} from 'angular2/testing';
+
+import {Component} from 'angular2/angular2';
+
+import {NgbProgressbar} from "./progressbar";
+
+describe('ng-progressbar', () => {
+  describe('UI logic', () => {
```

Comment:
Didn't touch tests yet, and I was thinking about business logic to test calculation yes.